docs/source/en/api/pipelines/kolors.md

```diff
@@ -41,6 +41,64 @@ image = pipe(
 image.save("kolors_sample.png")
 ```
 
+## IP Adapter
```

Comment:
I think it might be clearer to nest this under Usage Example (ie, IP-Adapter is a usage example and not something separate).

```suggestion
### IP Adapter
```